pyasic/miners/models/epic/blockminer/blockminer.py

```diff
@@ -0,0 +1,7 @@
+from pyasic.miners.makes import ePICMake
+
+
+class Blockminer520i(ePICMake):
+    raw_model = "Blockminer 520i"
```

Comment:
Tempted to call this `BlockMiner` instead, you can compare to other miner types and decide, but it should be consistent with the subclass...

Reply:
I'm indifferent and can change it to what you prefer.. If I said there was a "Blockminer 720i" and then a follow on "Blockminer2 XXX" does that change the thought process?

Reply:
I'm not sure we have a precedent for that yet.  Could make things complex...

I'm not opposed to having a `BlockMiner720i` and `BlockMiner2720i`, as long as they're distinct.

Even `BlockMiner1720i` isn't the worst thing, but I would much prefer to follow the API output/correct model information as much as possible.

I think honestly the manufacturer should just name them differently in the first place ;), and if I had my say on that one, I would actually have the 720i and the 1720i, use the last 4 characters as a sub model and the first one as a major version, similar to what Whatsminer does.